azext_iot/tests/test_iot_ext_int.py

```diff
@@ -71,6 +71,33 @@ def test_hub(self):
             expect_failure=True,
         )
 
+        # Test 'az iot hub connection-string show'
+        conn_str_pattern = r'^HostName={0}.azure-devices.net;SharedAccessKeyName=iothubowner;SharedAccessKey='.format(
+            LIVE_HUB)
+        conn_str_eventhub_pattern = r'^Endpoint=sb://'
+        self.cmd('iot hub connection-string show -n {0}'.format(LIVE_HUB), checks=[
+            self.check_pattern('connectionString', conn_str_pattern)
+        ])
+
+        self.cmd('iot hub connection-string show -n {0} --eh'.format(LIVE_HUB), checks=[
+            self.check_pattern('connectionString', conn_str_eventhub_pattern)
+        ])
+
+        self.cmd('iot hub connection-string show -n {0} -g {1}'.format(LIVE_HUB, LIVE_RG), checks=[
+            self.check('length(@)', 1),
+            self.check_pattern('connectionString', conn_str_pattern)
+        ])
+
+        self.cmd('iot hub connection-string show -n {0} -g {1} --all'.format(LIVE_HUB, LIVE_RG), checks=[
+            self.check('length(connectionString[*])', 5),
+            self.check_pattern('connectionString[0]', conn_str_pattern)
+        ])
+
+        self.cmd('iot hub connection-string show -n {0} -g {1} --all --eh'.format(LIVE_HUB, LIVE_RG), checks=[
+            self.check('length(connectionString[*])', 2),
```

Comment:
Same thoughts for checking the length of the connectionString list here. Also in the default eventhub case we can expect at least 1 policy by design.

Reply:
Done